src/core/json-schema-components.jsx

```diff
@@ -345,7 +373,8 @@ export class JsonSchema_boolean extends Component {
                     disabled={ disabled }
                     allowedValues={ enumValue }
                     allowEmptyValue={ !required }
-                    onChange={ this.onEnumChange }/>)
+                    onChange={ this.onEnumChange }
+                    { ...ariaProps } />)
```

Comment:
Can we fix the formatting on this `Select` block? e.g. `/>` and `)` seem to be incorrectly indented

Reply:
I believe that's the code style used by `swagger-ui`, they are still using this style in the upstream code